src/components/dashboard-widgets/dashboard-enum-widget/dashboard-enum-widget.component.html

```diff
@@ -0,0 +1,53 @@
+<ux-dashboard-widget #widget
+                     class="widget-element"
+                     [id]="id"
+                     [colSpan]="colSpan"
+                     [rowSpan]="rowSpan"
+                     [name]="name"
+                     [resizable]="!fixedMode">
+    <button *ngIf="!fixedMode"
+            class="widget-grab-handle"
+            uxDashboardGrabHandle
+            uxDashboardWidgetDragHandle>
+        <ux-icon name="pan"></ux-icon>
+    </button>
+    <h3 class="widget-title">{{ heading }}</h3>
+    <div class="widget-status">
+        <ng-container *ngIf="value | getEnumConfigByValue : enums : 'iconTemplate' as customIcon">
+            <ng-container *ngTemplateOutlet="customIcon"></ng-container>
+        </ng-container>
+        <ux-icon *ngIf="value | getEnumConfigByValue : enums : 'icon' as icon" name="{{ icon }}"></ux-icon>
+        {{ value | getEnumConfigByValue : enums : 'label' }}
+    </div>
+    <button class="widget-side-panel-opener" (click)="open()">
+        <ux-icon name="chevron-right"></ux-icon>
+    </button>
+</ux-dashboard-widget>
+<ux-side-panel #sidePanel
+               class="widget-side-panel"
+               [attachTo]="'container'"
+               [width]="'100%'">
+    <div class="ux-side-panel-header">
+        <h3>{{ heading }}</h3>
+        <button uxSidePanelClose
+                aria-label="Close Side Panel"
+                type="button"
+                class="btn btn-lg btn-link btn-icon button-secondary">
+            <ux-icon name="close"></ux-icon>
+        </button>
+    </div>
+    <ux-select-list #enumList
+                    class="ux-side-panel-content"
+                    [(selected)]="selected"
+                    (selectedChange)="changeSelection($event)">
```

Comment:
This will define (selectedChange) twice. Rather use [selected] and (selectedChange). In the latter, check if the selection is empty and assign only then.